.github/workflows/code_checks.yml

```diff
@@ -0,0 +1,51 @@
+name: code_checks
+on: [push]
+jobs:
+  linting:
+    runs-on: ubuntu-20.04
+    env:
+      BUNDLE_ENTERPRISE__CONTRIBSYS__COM: ${{ secrets.BUNDLE_ENTERPRISE__CONTRIBSYS__COM }}
+      RAILS_ENV: test 
+
+    services:
+      postgres:
+        image: postgis/postgis:11-2.5
+        env:
+          POSTGRES_USER: vets-api
+          POSTGRES_DB: vets-api_test
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+
+      redis:
+        image: redis:3.2-alpine
```

Comment:
We're on 5.0.4. We should at least use the same MAJOR here.

Reply:
Removed postgres and redis for the linting run